package.json

```diff
@@ -94,6 +94,7 @@
 		"docker-jetpack-cloud": "docker run -it --env CALYPSO_ENV=jetpack-cloud-production --name wp-calypso --rm -p 80:3000 wp-calypso",
 		"eslint-branch": "node bin/eslint-branch.js",
 		"feature-search": "node bin/feature-search.js",
+		"generate": "wp-generate",
```

Comment:
I feel like `wp-*` might not be a good name for the binary, since the packages aren't really related to WordPress